starter-code/index.html

```diff
@@ -10,5 +10,28 @@
   <title>Ironhack cart</title>
 </head>
 <body>
+    <header>
+        <div class= "container">
+            <h1>Ironhack<br>Merchandising Shop</h1>
+        </div>
+    </header>
+    <section class= "cart">
+      <script>createNewItem()</script>
+        <div id= "wrapper" class="container product wrapper">
+            <script>createItemNode()</script>
+            <div class= "btn delete">
+                <span> Delete </span>
+            </div>
+
+        </div>
+    </footer >
+
+       <div id= "footer" class= "container">
+
+          <button id= "calc-prices-button" class= "btn success" onclick= 'getTotalPrice(product.price, product.quantity)' >
```

Comment:
`onclick= 'getTotalPrice(product.price, product.quantity)'`
Is better if you do a `addEventListener` in the js file instead declare it in the HTML.